internal/service/alertconfiguration/resource_alert_configuration_test.go

```diff
@@ -423,25 +423,24 @@ func TestAccConfigRSAlertConfiguration_withPagerDuty(t *testing.T) {
 }
 
 func TestAccConfigAlertConfiguration_PagerDutyUsingIntegrationID(t *testing.T) {
-	acc.SkipTestForCI(t) // currently skipped as third party integration id cannot be created and obtained during test
-	proxyPort := replay.SetupReplayProxy(t)
-
+	// create a new project as it need to ensure no third party integration has already been created
 	var (
-		pagerDutyIntegrationID = os.Getenv("MONGODB_ATLAS_PAGER_DUTY_THIRD_PARTY_INTEGRATION_ID")
-		projectID              = replay.ManageProjectID(t, acc.ProjectIDExecution)
+		orgID       = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName = acc.RandomProjectName()
+		serviceKey  = dummy32CharKey
```

Comment:
replay logic is not handled here, can follow up with another PR as it will require some changes for what is capture in the env file.